cli_reference/basic_cli_operations.adoc

```diff
@@ -443,7 +443,7 @@ $oc start-build --no-cache
 ----
 
 === deploy
-View a deployment, or manually start, cancel, or retry a deployment:
+View a deployment, or manually start, or retry a deployment:
```

Comment:
@bfallonf this command is now deprecated:

Command "deploy" is deprecated, Use the `rollout latest` and `rollout cancel` commands instead.

I would probably just remove all mentions of it from docs at this point. WDYT?